README.md

```diff
@@ -1,24 +1,78 @@
-# README
+# Ruby capston : catalog to my things
```

Comment:
- [ ] Let's add a title that matches the project at hand, the title and the video do not represent the current project. This is a Rails project and not a Ruby capstone console app.

Reply:
oh sorry about that I did not edit it properly, I will make the changes now.thanks